rasa/test.py

```diff
@@ -110,6 +112,7 @@ def test_core(
     output: Text = DEFAULT_RESULTS_PATH,
     additional_arguments: Optional[Dict] = None,
 ) -> None:
+    """Checks if models are present and then runs the test function in the event loop."""
```

Comment:
I think it's better to describe what and why things are happening in a docstring instead of describing how things are done. 
```suggestion
    """Tests a trained Core model against a set of test stories."""
```

Reply:
Changed it! I will keep this advice also for all the future docstrings.